frameworks/Python/pyramid/frameworkbenchmarks/views.py

```diff
@@ -21,7 +21,7 @@ def test_2(request):
     """
     Test type 2: Single database query
     """
-    num = randint(1, 10001)
+    num = randint(1, 10000)
```

Comment:
I tried ```randint(0, 2)``` in python 2.7 and 3.4, and both versions gave results of 0, 1, and 2.  It looks like the other python tests were using ```randint(1, 10000)``` as well.